hack/util.sh

```diff
@@ -37,7 +37,7 @@ KARMADA_METRICS_ADAPTER_LABEL="karmada-metrics-adapter"
 
 KARMADA_GO_PACKAGE="github.com/karmada-io/karmada"
 
-MIN_Go_VERSION=go1.22.9
+MIN_Go_VERSION=go1.22.11
```

Comment:
1. The casing is a bit weird - should it be `MIN_GO_VERSION`? Maybe I can change it in a separate PR?
2. Is this version always in sync with the versions in `.go-version` and `go.mod`? How about just reading it from `.go-version`?

Reply:
Both are great to me! Thanks.